systemd/nexus-setup.sh

```diff
@@ -67,12 +67,7 @@ fi
 set -x
 
 if ! podman image inspect --type image "$NEXUS_SETUP_IMAGE" &>/dev/null; then
-    # load the image
     podman load -i "$NEXUS_SETUP_IMAGE_PATH" || exit
-    # get the image id
-    CRAY_NEXUS_SETUP_ID=$(podman images --noheading --format "{{.Id}}" --filter label="org.label-schema.name=cray-nexus-setup")
```

Comment:
This may not work properly, if there are more then 1 cray-nexus-setup images in podman. Anyway it's not needed anymore, because image comes already tagged after change in packaging.